src/stream.rs

```diff
@@ -283,6 +291,7 @@ where
     }
 }
 
+#[cfg(feature = "__decoder")]
```

Comment:
Without playback support this whole file should be excluded.
Does anyone actually need to exclude all features, including default ones?

Reply:
Generally I suggest as best practice that projects compile correctly with no default features. 

I ran into this myself, as I was upgrading a project that only uses the playback feature but none of the decoders.